specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -955,22 +961,22 @@ components:
           properties:
             beslagen:
               type: "array"
-              items: 
+              items:
                 $ref: "#/components/schemas/BeslagHal"
     Beslaglegger:
       allOf:
-        - description: "Dit element is opgenomen om de identificatie van de beslaglegger beschikbaar te stellen voor partijen die geen gebruik willen maken van de HalLinks voor een verwijzing naar een resource buiten het domein van deze API"
         - $ref: "#/components/schemas/ResourceIdentificatie"
-    Hypotheek: 
+        - description: "Dit element is opgenomen om de identificatie van de beslaglegger beschikbaar te stellen voor partijen die geen gebruik willen maken van de HalLinks voor een verwijzing naar een resource buiten het domein van deze API"
```

Comment:
In OAS 3.1 is het mogelijk om een description element als sibling te hebben van een $ref. Zie https://github.com/OAI/OpenAPI-Specification/issues/2033. We kunnen ervoor kiezen om hierop vooruit te lopen en de allOf te verwijderen en de description een sibling te laten zijn van de $ref. Code generatie in .NET gaat goed en je krijgt voor nu alleen een warning in SwaggerHub en Swagger Editor